cl_sii/dte/constants.py

```diff
@@ -388,3 +390,193 @@ def emisor_is_vendedor(self) -> bool:
+#   - XML element 'DTEDefType/Documento/Referencia/RUTOtr'
+#     - description: "RUT Otro Contribuyente"
+#     - XML type: 'SiiDte:RUTType'
+#     (https://github.com/cl-sii-extraoficial/archivos-oficiales/blob/master/src/code/rtc/2019-12-12-schema_cesion/schema_cesion/DTE_v10.xsd#L1804-L1808)
+# TODO: Use `TipoDocumentoEnum` after deprecate `TipoDteEnum`
+DTE_REFERENCIA_RUTOTR_TIPO_DOC_SET: FrozenSet[TipoDteEnum] = frozenset({
+    TipoDteEnum.NOTA_CREDITO_ELECTRONICA,
+    TipoDteEnum.NOTA_DEBITO_ELECTRONICA,
+    TipoDteEnum.FACTURA_COMPRA_ELECTRONICA,
+})
+
+
+###############################################################################
+# DTE Fields / "Referencia" / "Fecha de la Referencia"
+###############################################################################
+
+# The 'fecha_emision' of the document referred to.
+#
+# > Campo: Fecha de la Referencia
+# > Descripción: Fecha del documento de referencia.
+# > Tipo: ALFA
+# > Validación: Desde 2002-08-01 hasta 2050-12-31
+#
+#
+# Source:
+#   - Document "FORMATO DOCUMENTOS TRIBUTARIOS ELECTRÓNICOS (2019-11-15)" (retrieved on 2020-01-30)
+#     (https://github.com/cl-sii-extraoficial/archivos-oficiales/blob/master/src/docs/dte/formato-dtes/2019-11-15-formato-dtes-v2.2.y.pdf)
+#   - XML element 'DTEDefType/Documento/Referencia/FchRef'
+#     - description: "Fecha de la Referencia."
+#     - XML type: 'SiiDte:FechaType'
+#     (https://github.com/cl-sii-extraoficial/archivos-oficiales/blob/master/src/code/rtc/2019-12-12-schema_cesion/schema_cesion/DTE_v10.xsd#L1809-L1813)
+DTE_REFERENCIA_FECHA_NOT_BEFORE: date = date(2002, 8, 1)
+DTE_REFERENCIA_FECHA_NOT_AFTER: date = date(2050, 12, 31)
+
+
+###############################################################################
+# DTE Fields / "Referencia" / "Código de referencia"
+###############################################################################
+
+# The 'fecha_emision' of the document referred to.
+#
+# > Campo: Código de referencia
+# > Descripción: Fecha del documento de referencia.
+# > Tipo: NUM
+# > Validación: Requerido para 56, 61, 111, 112
+#
+# Source:
+#   - Document "FORMATO DOCUMENTOS TRIBUTARIOS ELECTRÓNICOS (2019-11-15)" (retrieved on 2020-01-30)
+#     (https://github.com/cl-sii-extraoficial/archivos-oficiales/blob/master/src/docs/dte/formato-dtes/2019-11-15-formato-dtes-v2.2.y.pdf)
+#   - XML element 'DTEDefType/Documento/Referencia/CodRef'
+#     - description: "Tipo de Uso de la Referencia."
+#     - XML type: 'xs:positiveInteger'
+#     (https://github.com/cl-sii-extraoficial/archivos-oficiales/blob/master/src/code/rtc/2019-12-12-schema_cesion/schema_cesion/DTE_v10.xsd#L1814-L1837)
+# TODO: Use `TipoDocumentoEnum` after deprecate `TipoDteEnum`
+DTE_REFERENCIA_CODREF_TIPO_DOC_MANDATORY_SET: FrozenSet[TipoDteEnum] = frozenset({
+    TipoDteEnum.NOTA_CREDITO_ELECTRONICA,
+    TipoDteEnum.NOTA_DEBITO_ELECTRONICA,
+})
+
+
+@enum.unique
+class CodigoReferenciaEnum(enum.IntEnum):
```

Comment:
Please remove the suffix `Enum` from `CodigoReferenciaEnum`. Perhaps you were inspired by `TipoDteEnum` but that one, authored by me, shouldn't have had the suffix as well.